pkg/microservice/aslan/core/common/service/registry/types.go

```diff
@@ -16,6 +16,10 @@ limitations under the License.
 
 package registry
 
+const (
+	SWRProvider = "swr"
```

Comment:
why this const is defined twice in two different place?

Reply:
remove it